controllers/wizard/fileUpload/form/SubmissionFilesUploadForm.inc.php

```diff
@@ -177,6 +177,11 @@ function execute() {
 				'username' => $user->getUsername()
 			)
 		);
+		
+		$hookResult = parent::execute($submissionFile);
```

Comment:
I would love to replace this pattern with something else -- but doing some experimentation with variadic parameter lists and references, it's a little inflexible (either all variadic parameters must be passed by reference using `&...$listName` or all must be passed by value without the `&`). I'll give this some thought but for the moment this will stick around.